components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -598,14 +606,54 @@ void RewardsServiceImpl::CreateRewardsWallet(
         observer.OnRewardsWalletUpdated();
       }
 
-      std::move(callback).Run(ledger::mojom::Result::LEDGER_OK);
+      std::move(callback).Run(result);
     };
 
     self->bat_ledger_->CreateRewardsWallet(
-        base::BindOnce(on_created, self, std::move(callback)));
+        country, base::BindOnce(on_created, self, std::move(country),
+                                std::move(callback)));
   };
 
-  StartProcess(base::BindOnce(on_start, AsWeakPtr(), std::move(callback)));
+  StartProcess(base::BindOnce(on_start, AsWeakPtr(), std::string(country),
```

Comment:
I'm not sure I understand why `std::string(country)` is needed — `country` by itself should work.

Reply:
Fixed